package.json

```diff
@@ -22,7 +22,7 @@
     "test": "turbo test",
     "build": "turbo build",
     "check": "turbo check",
-    "lint": "turbo lint --concurrency=4",
+    "lint": "NODE_OPTIONS='--max-old-space-size=4096' turbo lint --concurrency=2",
```

Comment:
Unrelated, but linter tests are still failing after https://github.com/pagopa/interop-be-monorepo/pull/632